msteams-platform/samples/virtual-table-entity-references.md

```diff
@@ -0,0 +1,292 @@
+|`m365_reminderminutesbeforestart`|IntegerType|The number of minutes before the event start time that the reminder alert occurs.|
+|`m365_responserequested`|BooleanType|Default is true, which represents the organizer would like an invitee to send a response to the event.|
+|`m365_responsestatus`|StringType|Indicates the type of response sent in response to an event message. JSON encoded string, max 4000 in length.{\"response\": \"String\",\"time\": \"String (timestamp)\"}|
+|`m365_sensitivity`|StringType|Possible values are: normal, personal, private, confidential.|
+|`m365_seriesmasterid`|StringType|The ID for the recurring series master item, if this event is part of a recurring series.|
+|`m365_showas`|StringType|The status to show. Possible values are: free, tentative, busy, oof, workingElsewhere, unknown.|
+|`m365_subject`|StringType|The text of the event's subject line. Primary lookup column|
+|`m365_transactionid`|StringType|A custom identifier specified by a client app for the server to avoid redundant POST operations in case of client retries to create the same event. This is useful when low network connectivity causes the client to time out before receiving a response from the server for the client's prior create-event request. After you set `transactionId` when creating an event, you can't change transactionId in a subsequent update. This property is only returned in a response payload if an app has set it. Optional.|
+|`m365_type`|StringType|The event type. Possible values are: singleInstance, occurrence, exception, seriesMaster. Read-only|
+|`m365_weblink`|StringType|The URL to open the event in Outlook on the web.Outlook on the web opens the event in the browser if you're signed in to your mailbox. Otherwise, Outlook on the web prompts you to sign in. This URL cannot be accessed from within an iFrame.|
+|`m365_grapheventid`|StringType|Unique identifier of the graph event.|
+|`m365_groupid`|StringType|Group ID to which the event belongs.|
+
+## Graph Booking Appointment
+
+* Entity name: m365_graphbookingappointment
+* Graph resource: [bookingAppointment resource type](/graph/api/resources/bookingappointment)
+* Sorting isn't supported.
+* Filtering is supported on the following columns:
+  * m365_bookingbusinessID
+  * m365_collaborationrootID
+  * m365_customertimezone
+  * m365_optoutofcustomeremail
+  * m365_price
+  * m365_pricetype
+  * m365_serviceID
+  * m365_servicename
+* Pagination isn't supported.
+
+### Attributes for Graph Booking Appointment
+
+| Column  | Dataverse Type | Details |
+|---|---|---|
+| `m365_collaborationrootid`| StringType| Collaboration root ID of the collaboration session the record is associated with. If the record is associated with multiple collaboration sessions this will be returned as a comma delimited string. Note that this attribute won't be returned when retrieving multiple records.|
+| `m365_graphbookingappointmentid` | StringType | Unique Identifier of the graph booking appointment.|
```

Comment:
Change 

| `m365_graphbookingappointmentId` | guid | Unique Identifier of the graph booking appointment. |